library/std/src/panic.rs

```diff
@@ -114,6 +114,9 @@ where
 /// aborting the process as well. This function *only* catches unwinding panics,
 /// not those that abort the process.
 ///
+/// If a custom panic hook has been set, note that it will be invoked before
+/// the panic is caught.
```

Comment:
Let's also mention that the hook is executed before _unwinding_ starts:

```suggestion
/// Note that if a custom panic hook has been set, it will be invoked before
/// the panic is caught, before unwinding.
```